app/preprints/-components/submit/file/upload-file/template.hbs

```diff
@@ -1,4 +1,4 @@
-{{#if this.prepUrl.isRunning}}
```

Comment:
This was preventing some problems on initialization where the buildURL was waiting for things to be read from the server. You probably want to `or` these two if you also need `success`.